src/HotChocolate/ApolloFederation/src/ApolloFederation/ExternalDirectiveType.cs

```diff
@@ -1,43 +1,30 @@
-using System.Collections.Generic;
-using System.Linq;
 using HotChocolate.ApolloFederation.Properties;
-using HotChocolate.Configuration;
 using HotChocolate.Types;
 
 namespace HotChocolate.ApolloFederation
 {
     /// <summary>
     /// The @external directive is used to mark a field as owned by another service.
-    /// This allows service A to use fields from service B while also knowing at
-    /// runtime the types of that field.
+    /// This allows service A to use fields from service B while also knowing at runtime
+    /// the types of that field.
+    ///
+    /// <example>
+    /// # extended from the Users service
```

Comment:
I added the examples from the spec.

Reply:
https://github.com/ChilliCream/hotchocolate/pull/2372#discussion_r496732474